documentation/assemblies/configuring/assembly-config-kafka.adoc

```diff
@@ -6,14 +6,14 @@
 = Kafka cluster configuration
 
 [role="_abstract"]
-This section describes how to configure a Kafka deployment in your Strimzi cluster.
-A Kafka cluster is deployed with a ZooKeeper cluster. The deployment can also include the Topic Operator and User Operator, which manage Kafka topics and users.
-
-You configure Kafka using the `Kafka` resource.
-Configuration options are also available for ZooKeeper and the Entity Operator within the `Kafka` resource.
+Configure a Kafka deployment using the `Kafka` resource.
+A Kafka cluster is deployed with a ZooKeeper cluster. 
+A deployment can also include the Topic Operator and User Operator, which manage Kafka topics and users.
+Configuration options are available for ZooKeeper and the Entity Operator within the `Kafka` resource.
```

Comment:
I think we should make clear that the EO section in the Kafka resource is where you configure the TO and UO mentioned before.

Reply:
Okay. I've rewritten to make clear

> Configure a Kafka deployment using the `Kafka` resource.
> A Kafka cluster is deployed with a ZooKeeper cluster, so configuration options are also available for ZooKeeper within the `Kafka` resource.
> You can also configure `entityOperator` properties in the `Kafka` resource to include the Topic Operator and User Operator in the deployment. 
> The Entity Operator comprises the Topic Operator and User Operator.